src/window.rs

```diff
@@ -682,6 +682,28 @@ impl Window {
     pub fn set_ime_position<P: Into<Position>>(&self, position: P) {
         self.window.set_ime_position(position.into())
     }
+
+    /// Requests user attention to the window, this has no effect if the application
+    /// is already focused.
```

Comment:
You should document what `None` does, also should we do that via `None` or via special method, like `unset_user_attention` or similar.

I wonder what others think about it, @chrisduerr , @alvinhochun , @francesca64 ?

Reply:
I tried to stay consistent with other APIs like `set_window_icon`, which when providing `None` calls an `unset` method internally. It's both consistent and more ergonomic than providing a separate `unset_user_attention` I believe.

Reply:
I think we can go with `None` then.